LICENSE update: Add MIT license for infrastructure / scripts (#1593)
galaxyproject/training-material: README.md

# License

> This content of this work is licensed under the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0).

Reviewer: ```suggestion
The content of the tutorials and website are licensed under the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0).
```

maybe? Just want to be a bit more explicit.
